File: sdk/scala-gatling/src/gatling/scala/org/openapitools/client/model/InvalidCode.scala

package org.openapitools.client.model


case class InvalidCode (
    _message: Option[String],
    _status: Option[Integer]
)
object InvalidCode {
    def toStringBody(var_message: Object, var_status: Object) =
        s"""
        | {
        | "message":$var_message,"status":$var_status
        | }
        """.stripMargin
}

File: sdk/kotlin-spring/src/main/kotlin/org/openapitools/model/InvalidCode.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size

/**
 * 
 * @param message 
 * @param status 
 */
data class InvalidCode(

    @JsonProperty("message") val message: kotlin.String? = null,

    @JsonProperty("status") val status: kotlin.Int? = null
) {

}

